### DISCUSSION TITLE
OLEDのジャンパ不良の代替

### DISCUSSION THREAD

**Discussion by akatako** [Yowkees/keyball#266] [Q&A] [answered]
Keyball61(rev2)の左手用基盤（ボールがつかない方の基盤）についてなのですが、半田付け初心者ということもあり、右から2つ目のジャンパの半田付けが難しい状態になっています（フラッスクの塗布やヤニの除去などはおこないましたが、十分に温めてもハンダを弾いてしまっています。）
Q＆Aは一通り眺めましたが、ProMicroを取り付ける部分？の印字が下記urlとは違うため、同じ場所でショートさせても上手く動作しないだろうと考えています。
適切なショート箇所をご教授いただけないでしょうか？
https://github.com/Yowkees/keyball/discussions/187#discussion-4783204

![IMG_3391](https://github.com/Yowkees/keyball/assets/54677420/c1432764-9c32-4eb5-a82d-58a33fb1a477)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。了解しました。
下記画像の赤点２点をエナメル線等で接続していただくと、ジャンパーしたのと同じ回路になります。
なお、この作業は４Ｐピンソケットを半田付けしてから行ってください。
もし基板パッドへの半田付けが難しそうでした、赤丸とオレンジ丸を接続いでいただいても同じ回路です。
以上、お手数ですがご対応お願いいたします。

![KB61L_jampfix](https://github.com/Yowkees/keyball/assets/84209772/36f24ece-a418-4477-82d3-579944cee62c)

**Comment by akatako** [comment]
@Yowkees 
ご回答ありがとうございます！
おかげで動作確認することが出来ました！
ありがとうございます！
